techniques/css/C28.html

```diff
@@ -28,33 +28,33 @@ <h2>Description</h2>
 	<section id="examples">
 		<h2>Examples</h2>
 		<section class="example">
-			<h3><code class="language-css">em</code> units for sizes for layout container containing text </h3>
+			<h3><code>em</code> units for sizes for layout container containing text </h3>
 
-			<p>In this example, a <code class="language-html">div</code> element, with <code class="language-html">id</code> value of <code class="language-html">"nav_menu"</code>, is used to position the navigation menu along the left-hand side of the main content area of the web page. The navigation menu consists of a list of text links, with <code class="language-html">id</code> value of <code class="language-html">"nav_list"</code>. The text size for the navigation links and the width of the container are specified in <code class="language-css">em</code> units.</p>
+			<p>In this example, a <code>div</code> element, with <code>id</code> value of <code>"nav_menu"</code>, is used to position the navigation menu along the left-hand side of the main content area of the web page. The navigation menu consists of a list of text links, with <code>id</code> value of <code>"nav_list"</code>. The text size for the navigation links and the width of the container are specified in <code>em</code> units.</p>
```

Comment:
that id value may benefit from highlighting (or even rewrite as `id="nav_list"` with some wordsmithing, and then do highlighting)